webcompat/templates/new-issue.html

```diff
@@ -1,9 +1,5 @@
 {%- extends "layout.html" -%}
 
-{% block extracss %}
-<link href="{{ url_for('static', filename='css/src/issue-wizard.css') }}" type="text/css" rel="stylesheet">
```

Comment:
Is this unused?

Reply:
It has moved to https://github.com/webcompat/webcompat.com/blob/4bb999e2c08cd9629723912576e7174ed2c90dfb/webcompat/static/js/lib/webcompat.js , since css files need to be included in js in order for webpack to pick it up

Reply:
Aha, thanks. Let's pretend I already noticed that, since I reviewed it. 🙈